core/src/main/java/org/apache/iceberg/encryption/PlaintextEncryptionManager.java

```diff
@@ -25,6 +25,8 @@
 import org.slf4j.LoggerFactory;
 
 public class PlaintextEncryptionManager implements EncryptionManager {
+  public static final EncryptionManager INSTANCE = new PlaintextEncryptionManager();
```

Comment:
I think we typically hide these behind static `instance` methods, rather than making the instance field public.